trp_external_user/tests/test_trp_external_user.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp.exceptions import AccessError
```

Comment:
From odoo.... import instead of from openerp....

Reply:
thanks, done